### DISCUSSION TITLE
在提交issue的时候就填写识别码

### DISCUSSION THREAD

**Discussion by ghost** [Meloong-Git/PCL#993] [提问] [answered]
在提交issue的时候就填写识别码，这样可以issue在标记为完成的时候就自动解锁活跃橙主题(只是问一下，如果可以的话我就提交issue)

**Comment by LTCatt** [comment, marked as answer]
有的人用的是正式版，没法使用主题功能，也有人是已经解锁了……如果要求填写识别码反而很麻烦，不考虑……
